app/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ apply(from = "${project.rootDir}/jacoco/jacoco.gradle.kts")
 repositories {
     maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots") }
     mavenCentral()
+    mavenLocal()
```

Comment:
Is this necessary?

Reply:
Not at all. I added to be able to use the mobile ui before publishing. I'll remove it now :)